README.md

```diff
@@ -88,87 +25,69 @@ sierra = true
-
-```bash
-cargo run --bin starknet-contract-verifier
 ```
 
-If you are using `asdf` for the management of scarb binary on a project basis, you should make sure that the verifier runs in the directory of the project so that the verifier will detect and use the correct `scarb` binary for that project.
+### Setup rust
 
-You can build the binaries and add it to path to make it easier to use the verifier.
-
-```bash
-# build all binaries
-cargo build --all --release
-
-# then add build target directory to path 
-# depending on your shell this might be different.
-# Add the following to the end of your shell configuration file
-export PATH="$PATH:/path/to/starknet-contract-verifier/target/release"
-
-# you should now be able to call the verifier directly if build succeeds.
-starknet-contract-verifier
-```
-
-You should be greeted with prompts that asks for the details of your cairo project & contracts, and will be guided step by step through the verification process.
-
-## Building from source
-
-If you are developing and building the project from source, you will first need to install rust.
+`starknet-contract-verifier` is a rust/cargo project. In order to build it you'll need rust and cargo set up. You can do it easily using [rustup](https://rustup.rs/).
 
 ```bash
 curl https://sh.rustup.rs -sSf | sh -s
 ```
 
-> Note: Builds for 2.4.3 and below only works with < Rust 1.77. As such please make sure that you have the correct rust version before building.
+### Submit your contract
 
-To build the project, simply do
+You are good to go, execute:
 
-```bash
-cargo build
+``` bash
+cargo run -- --network mainnet submit \
+  --name <YOUR_CONTRACT_NAME> \
+  --hash <YOUR_CONTRACT_CLASS_HASH> \
+  --path <PATH_TO_YOUR_SCARB_PROJECT>
 ```
 
-and the project should start building.
+When successful you'll be given verification job id, which you can pass to:
 
-## Limitations and Known Issues
+``` bash
+cargo run -- --network mainnet status --job <JOB_ID>
+```
 
-There's a few known issue with the current implementation of this verifier.
+to check the verification status. Afterwards visit [Voyager website]() and search for your class hash to see *verified* badge.
 
+## Detailed information
 
-### 1. Rearranging and restructure of the resulting verified files
+### Verification
 
-It's possible that your modules get re-arranged after verification as the verifier tries to resolve your dependencies from the main contract and re-generate them into a new project. This is known and we are working towards making the generated project look as closely as possible with the original module structure.
+`starknet-contract-verifier` provides two subcommands: `submit` and `status`. For both cases user needs to select the network with which they want to interact via the `--network` argument. Possible cases are:
+- `mainnet`, main starknet newtwork,
+- `testnet`, sepolia test networ,
```

Comment:
I think it is better to also use `sepolia` for the network name here.
It is quite common, and there is no ambiguity when testnet name changes (like the goerli-> sepolia transition)